src/Raven.Client/Raven.Client.csproj

```diff
@@ -72,4 +72,7 @@
     <PackageReference Include="System.Threading.ThreadPool" Version="4.3.0" />
     <PackageReference Include="System.ValueTuple" Version="4.5.0" />
   </ItemGroup>
+  <ItemGroup Condition="$(TargetFramework) == 'netcoreapp2.1'">
+    <PackageReference Include="Microsoft.AspNetCore.WebSockets" Version="2.1.1" />
```

Comment:
This is a server-side package? Why do we need this?

Reply:
Yes you are right, it's not required